golem_core/payload/base.py

```diff
@@ -24,15 +22,15 @@ class BasePayload(AutodecoratingModel, ABC):
         @dataclass
         class MyPayload(BasePayload):
             myprop: str = prop(CUSTOM_PROPERTY, default="myvalue")
-            runtime: str = constraint(inf.INF_RUNTIME_NAME, default=CUSTOM_RUNTIME_NAME)
-            min_mem_gib: float = constraint(inf.INF_MEM, operator=">=", default=16)
-            min_storage_gib: float = constraint(inf.INF_STORAGE, operator=">=", default=1024)
+            runtime: str = constraint("golem.runtime.name", default=CUSTOM_RUNTIME_NAME)
+            min_mem_gib: float = constraint("golem.inf.mem.gib", ">=", default=16)
+            min_storage_gib: float = constraint("golem.inf.storage.gib", ">=", default=1024)
```

Comment:
`golem.inf.storage.gib` seems like random value. How can I tell what are possible `constraint` keys? 

`inf.INF_RUNTIME_NAME, inf.INF_*` were at least telling that.

Reply:
Added.